### ISSUE TITLE
Divide By Zero error with Vertex.php

### ISSUE THREAD

**Issue by stevendesu** [thephpleague/geotools#94]
Attempting to find the bearing between two points that lie along the same line of latitude yields a divide by zero error.

Sample code:

```php
<?php

$geotools = new \League\Geotools\Geotools();
$coordA   = new \League\Geotools\Coordinate\Coordinate([10, 20]);
$coordB   = new \League\Geotools\Coordinate\Coordinate([10, 30]);
$vertex    =  $geotools->vertex()->setFrom($coordA)->setTo($coordB);
```

Expected output:

A vertex should be instantiated with a bearing of 0 degrees and final bearing of 0 degrees

Actual output:

```
ErrorException in Vertex.php line 76:
Division by zero
```

The code in question triggering the issue:

```php
// Vertex.php, Line 76
$this->gradient = ($this->to->getLongitude() - $this->from->getLongitude()) / ($this->to->getLatitude() - $this->from->getLatitude());
```

Note that the gradient and ordinate intercept are **only** used in the `isOnSameLine` function, and this function seems to properly handle the case of not having a valid gradient (there's some "is_null" stuff at the top). However the value is not getting set to null, and is instead throwing an exception.

**Comment by stevendesu** [comment]
I created a PR to fix this but I'm getting an error in `/home/travis/build/thephpleague/geotools/tests/TestCase.php` -- a file I didn't even touch. I think the test suite may have been broken by a recent commit.

I'm not sure where to start with diagnosing the problem

**Comment by Nyholm** [comment]
Fixed by #95
